tests/remote/remote_target.pm

```diff
@@ -1,34 +1,34 @@
 # SUSE's openQA tests
 #
-# Copyright © 2016 SUSE LLC
+# Copyright © 2016-2018 SUSE LLC
 #
 # Copying and distribution of this file, with or without modification,
 # are permitted in any medium without royalty provided the copyright
 # notice and this notice are preserved.  This file is offered as-is,
 # without any warranty.
 
-# Summary: Remote ssh & vnc installation
-# Maintainer: mkravec <mkravec@suse.com>
+# Summary: Controller/master for remote installations
+# Tags: poo#9576
+# Maintainer: Martin Loviska <mloviska@suse.com>
 
 use base "opensusebasetest";
 use strict;
 use testapi;
 use lockapi;
 use mmapi;
+use mm_network;
 
 # poo#9576
 sub run {
-    # Notice MASTER system is ready for installation
-    assert_screen "remote_slave_ready", 200;
-    mutex_create "installation_ready";
-
-    # Wait until MASTER finishes installing the system
-    wait_for_children;
+    assert_screen("remote_slave_ready", 350);
+    mutex_create("ssh_server_is_available");
+    # wait while whole installation process finishes
+    assert_screen("grub2", 1200);
```

Comment:
This is something that should be improved. We have better methods for that, e.g. the [wait_boot](https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/lib/opensusebasetest.pm#L291) method. Please try to use that. It has a good documentation about the supported options.

Reply:
I really appreciate that you have introduced to me to wait_boot function. Thanks!